assets/pages/bundles-landing/bundlesLanding.jsx

```diff
@@ -39,17 +43,24 @@ if (participations.addAnnualContributions) {
 
 // ----- Render ----- //
 
-const content = (
-  <Provider store={store}>
-    <div>
-      <SimpleHeader />
-      <Introduction />
-      <Bundles />
-      <WhySupport />
-      {store.getState().common.campaign === 'baseline_test' ? '' : <WaysOfSupport />}
-      <LinksFooter />
-    </div>
-  </Provider>
-);
+let content = null;
+
+// New Design Test: Replace with check for variant when test goes live.
+if (getQueryParameter('newDesigns', 'false') === 'true') {
+  content = supportLanding(store);
+} else {
+  content = (
+    <Provider store={store}>
+      <div>
+        <SimpleHeader />
+        <Introduction />
+        <Bundles />
+        <WhySupport />
+        {store.getState().common.campaign === 'baseline_test' ? '' : <WaysOfSupport />}
```

Comment:
Since you are here... we can delete this ternary check since we are not currently running this test.

Reply:
https://trello.com/c/kOUHpkfq/1035-delete-baseline-test-from-support-frontend